client/overview/index.js

```diff
@@ -144,27 +149,83 @@ const OverviewPage = () => {
 					) }
 				</Notice>
 			) }
-			<TestModeNotice
-				currentPage="overview"
-				isDevMode={ isDevMode }
-				actions={
-					isDevMode
-						? [
-								{
-									label: strings.notice.actions.setUpPayments,
-									onClick: () =>
-										setLivePaymentsModalVisible( true ),
-								},
-								{
-									label: strings.notice.actions.learnMore,
-									url:
-										'https://woocommerce.com/document/woopayments/testing-and-troubleshooting/sandbox-mode/',
-									urlTarget: '_blank',
-								},
-						  ]
-						: []
-				}
-			/>
+			{ isDevMode ? (
+				<BannerNotice status="warning" isDismissible={ false }>
```

Comment:
The notice is spanning too many lines in the Overview page file. IMO, we should extract it as a separate component and import it back here, like `TestModeNotice`.

Reply:
I thought about it but as with PHP abstractions I think we should first have the need to use this in multiple places before abstracting it. As it stands, this notice style is only for the overview page. That is why I left it inline. WDYT?

Reply:
That's a fair argument. I am usually not pedantic about having all components in separate files. However, in this case, it is embedded inside a ternary operator, making it hard to see the alternate case without collapsing it in the code editor. Besides, it's the only compound component in the file that is this, making things look a little inconsistent. WDYT about extracting it in the same file, same as `OverviewPageError`?

That said, I'll leave it up to you as it's a matter of preference.

Reply:
@anu-rock You do make a good argument. I will separate that code in the same file, under a function.